test/unit/collect_active_set.c

```diff
@@ -79,41 +114,30 @@ int main(void)
 
     /* A total of npes tests are performed, where the active set in each test
      * includes PEs i..npes-1 and each PE contributes PE ID elements */
-    for (i = 0; i <= me; i++) {
-        int j, k;
-        int idx = 0;
-
+#ifdef ENABLE_DEPRECATED_TESTS
+    for (i = 0; i < me; i++) {
```

Comment:
```suggestion
    for (i = 0; i <= me; i++) {
```
Probably a typo here...  I think this will deadlock if `ENABLE_DEPRECATED_TESTS` is turned on.